api/hypershift/v1beta1/azure.go

```diff
@@ -28,11 +28,8 @@ type AzureNodePoolPlatform struct {
 	// For example, "D32ads_v5" would be a suitable general purpose VM size, or "ND96_MI300X_v5" would represent a GPU accelerated VM.
 	//
 	// +kubebuilder:validation:Pattern=`^(Standard_|Basic_)?[A-Z]+[0-9]+(-[0-9]+)?[abdilmptsCNP]*(_[A-Z]*[0-9]+[A-Z]*)?(_v[0-9]+)?$`
-	// +kubebuilder:validation:Required
-	// + Azure VM size format described in https://learn.microsoft.com/en-us/azure/virtual-machines/vm-naming-conventions
```

Comment:
I think L32-34 is helpful info. Was this manually removed or did the linter remove it?